test/simple-wallet.test.ts

```diff
@@ -53,8 +53,9 @@ describe('SimpleAccount', function () {
 
   it('should pack in js the same as solidity', async () => {
     const op = await fillUserOpDefaults({ sender: accounts[0] })
+    const encoded = encodeUserOp(op)
     const packed = packUserOp(op)
-    expect(await testUtil.packUserOp(op)).to.equal(packed)
+    expect(await testUtil.packUserOp(packed)).to.equal(encoded)
```

Comment:
took me a white to read this...
clearly, need to rename the solidity helper too, its `testUtils.encodeUserOp`